core/shared/src/main/scala/zio/package.scala

```diff
@@ -80,4 +80,6 @@ package object zio
   val Hub: ZHub.type = ZHub
 
   type Semaphore = stm.TSemaphore
+
+  type HasMany[K, A] = Has[_ <: Map[K, A]]
```

Comment:
We have to find a way to test that use of a type wildcard here doesn't cause unsoundness issues with type tags.

Reply:
In fact with a type alias approach, maybe just `Has[Map[K, A]]` is better, since it is in fact a map.

Reply:
Yes I think that would help.